src/main/java/spoon/reflect/visitor/ImportScannerImpl.java

```diff
@@ -207,9 +172,6 @@ public void scan(CtElement element) {
 
 	@Override
 	public Collection<CtReference> computeAllImports(CtType<?> simpleType) {
-		classImports.clear();
-		fieldImports.clear();
-		methodImports.clear();
```

Comment:
I added them by security if computeAllImports was called multiple times with the same instance.